src/AstGen.zig

```diff
@@ -6167,6 +6167,10 @@ fn ret(gz: *GenZir, scope: *Scope, node: ast.Node.Index) InnerError!Zir.Inst.Ref
     const node_datas = tree.nodes.items(.data);
     const node_tags = tree.nodes.items(.tag);
 
+    if (astgen.fn_block == null) {
+        return astgen.failNode(node, "invalid 'return' outside function scope", .{});
```

Comment:
"invalid" is redundant since it's a compile error; the fact that something invalid happened is implied:

```suggestion
        return astgen.failNode(node, "'return' outside function scope", .{});
```

Reply:
I copied this from `invalid 'try' outside function scope`. Should that be changed to `'try' outside function scope` also? Also that is never tested, so I could add a test for that.

Reply:
Sounds good thank you!

Reply:
It is done :)